webclient/app/src/app/commons/appTemplateDialog/AppTemplateAuthDialog.jsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from "react";
+import { Button, TextField, Container, DialogTitle, Typography, IconButton, Box } from '@mui/material';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import { useTranslation } from 'react-i18next';
+import AppTemplateRest from "../../services/AppTemplateRest";
+import { CloudSync, Close } from "@mui/icons-material";
+import ErrorAlert from "../alert/ErrorAlert";
+import AppTemplateDialogStyles from "./AppTemplateDialogStyles";
+import NotificationDialog from "../alert/NotificationDialog";
+
+function AppTemplateAuthDialog(props) {
+    const { appTemplate, handleRefresh, setOpenSuccessDialog } = props;
+    const appTemplateDialogStyles = AppTemplateDialogStyles();
+    const [downloadRequestData, setDownloadRequestData] = useState({ "appTemplateId": null, "username": "", "password": "" });
+    const [alert, setAlert] = useState({"open":false, "title": "ERROR", "message": ""});
+    const { t } = useTranslation();
+
+    const appTemplateRest = new AppTemplateRest();
+
+    const [openAuthDialog, setOpenAuthDialog] = useState(false);
+  
+
+    const handleChange = (event) => {
+        const { name, value } = event.target;
+        let downloadRequestDataNew = { ...downloadRequestData };
+        downloadRequestDataNew[name] = value;
+        setDownloadRequestData(downloadRequestDataNew);
+    }
+
+    const handleLogin = () => {
+        console.log("in handleLogin");
+        if (appTemplate.credentialsRequired) {
+            handleAlertClose();
+            setOpenAuthDialog(true);
+        } else {
+            handleAppTemplateReload();
+        }
+    }
+
+    const handleAppTemplateReload = () => {
+        downloadRequestData.appTemplateId = appTemplate.id;
+        appTemplateRest.updateTemplates(downloadRequestData).then(() => {
+            handleRefresh();
+            setOpenAuthDialog(false);
+            setOpenSuccessDialog(true);
```

Comment:
Why in a different Dialog? A Snackbar (https://mui.com/components/snackbars/) would be better suited here I think

Reply:
Ticket ab#43 erstellt für Snackbar.